.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "submodules/non-existent-path"]
     path = submodules/non-existent-path
     url = git@github.com:pypa/sampleproject.git
+[submodule "relative-url-submodule"]
+	path = relative-url-submodule
+	url = ./submodules/relative-url-submodule.git
```

Comment:
```suggestion
[submodule "submodules/relative-url-submodule"]
	path = submodules/relative-url-submodule
	url = ../test-fixture-vcs-repository.git
```

Reply:
You'll need to make sure that the submodule ref is pinned to the branch I created for you as well.